doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,553 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "In this tutorial, you will learn how to do statistical analysis of your simulation data.\n",
+    "This is an important topic, because the statistics of your data determine how precise your simulation result is. Furthermore, knowing about the statistics can help you optimize your disk space usage.\n",
+    "\n",
+    "ESPResSo provides a lot of ways to take measurements of your system. Usually, you will sample a quantity many times during a simulation and in the end average over all samples. Intuitively, the simulation result will be more precise the more samples are taken during the simulation. However, this is not the whole truth. There are some things that need to be considered, which we will cover in this tutorial.\n",
+    "\n",
+    "Formally, if you determine a physical quantity by averaging over several samples, you only *approximate* the unknown, true mean value. Usually, the quantity is expected to fluctuate around its mean; therefore, you can never directly measure the mean. You are bound to take repeated measurements and in the end average over all samples (a finite number). In your report, you will present this average as your result. Additionally, you should express the precision of your measurements to give a proper meaning to your result. And this is where things get more involved.\n",
+    "\n",
+    "There are several different ways to express the precision of your measurements. We will begin by briefly discussing what they are and what their differences are. After that, we will continue with the *standard error of the mean* as a viable option to be presented in your simulation results.\n",
+    "\n",
+    "### Standard deviation\n",
+    "The standard deviation is a measure for how much individual samples are expected to deviate from the mean. We want to use precise terminology, and therefore need to state that, in fact, we cannot directly measure the standard deviation but only estimate it. A commonly used estimator for the standard error is\n",
+    "\n",
+    "$\n",
+    "\\begin{align}\n",
+    "    \\bar\\sigma = \\sqrt{\\frac{1}{N-1}\\sum_{i=1}^{N}(x_i-\\bar{x})^2}\\tag{1}\n",
+    "\\end{align}\n",
+    "$\n",
+    "\n",
+    "where $\\bar\\sigma$ is the estimator of the standard deviation $\\sigma$, $N$ the number of samples, $x_i$ the individual samples and $\\bar{x}$ their mean. This estimator somewhat resembles the \"square root of the variance\". However, instead of dividing by $N$ as for the variance, we divide by $N-1$. This accounts for the fact that we do not know the true mean value, but only estimate it (by averaging over all samples); and therefore we have one additional degree of freedom. Another way of viewing the $N-1$ divisor is to understand that for a single measurement, it is impossible to define the variance, which is represented by a \"division by zero\".\n",
```

Comment:
You are using <img src="https://render.githubusercontent.com/render/math?math=%5Coverline%7B%5Csigma%7D" alt="\overline{\sigma}" height=15> to refer to an estimator and <img src="https://render.githubusercontent.com/render/math?math=%5Coverline%7Bx%7D" alt="\overline{x}" height=15> to refer to an average. Textbooks usually one of these notations for estimators: <img src="https://render.githubusercontent.com/render/math?math=%5Chat%7B%5Csigma%7D%2C%20%5Csigma_%7B%5Coverline%7BX%7D%7D" alt="\hat{\sigma}, \sigma_{\overline{X}}" height=25> with *X* instead of *x* for the variable (*X* is the random process). A realization of an estimator has a different symbol, typically the latin version of the greek symbol, in this case *s*.